docs/fuse-box/tutorials/api-trade-quotes.mdx

```diff
@@ -0,0 +1,338 @@
+                    <tbody className="text-gray-700">
+                    <tr>
+                        <td className="border px-4 py-2">Source</td>
+                        <td className="border px-4 py-2">{order.source}</td>
+                    </tr>
+                    <tr>
+                        <td className="border px-4 py-2">Maker Token</td>
+                        <td className="border px-4 py-2">{order.makerToken}</td>
+                    </tr>
+                    <tr>
+                        <td className="border px-4 py-2">Taker Token</td>
+                        <td className="border px-4 py-2">{order.takerToken}</td>
+                    </tr>
+                    <tr>
+                        <td className="border px-4 py-2">Maker Amount</td>
+                        <td className="border px-4 py-2">{order.makerAmount}</td>
+                    </tr>
+                    <tr>
+                        <td className="border px-4 py-2">Taker Amount</td>
+                        <td className="border px-4 py-2">{order.takerAmount}</td>
+                    </tr>
+                    <tr>
+                        <td className="border px-4 py-2">Router</td>
+                        <td className="border px-4 py-2">{order.fillData.router}</td>
+                    </tr>
+                    <tr>
+                        <td className="border px-4 py-2">Token Address Path</td>
+                        <td className="border px-4 py-2">{order.fillData.tokenAddressPath}</td>
+                    </tr>
+                    <tr>
+                        <td className="border px-4 py-2">Uniswap Path</td>
+                        <td className="border px-4 py-2">{order.fillData.uniswapPath}</td>
+                    </tr>
+                    <tr>
+                        <td className="border px-4 py-2">Input</td>
+                        <td className="border px-4 py-2">{order.fill.input}</td>
+                    </tr>
+                    <tr>
+                        <td className="border px-4 py-2">Output</td>
+                        <td className="border px-4 py-2">{order.fill.output}</td>
+                    </tr>
+                    <tr>
+                        <td className="border px-4 py-2">Adjusted Output</td>
+                        <td className="border px-4 py-2">{order.fill.adjustedOutput}</td>
+                    </tr>
+                    </tbody>
+                </table>
+                </div>
+            ))}
+    </div>
+)}
+```
+
+Save the file.
+
+Run the application using the command:
+
+```bash
+npm run dev
+```
+
+## Conclusion
+This tutorial covered how to consume the Trade v2 API **”Get a quote for buying or selling any ERC20 token”** endpoint. This basic framework can be expanded to build more complex applications by integrating the other Trade v2 API Endpoints
+
+Checkout the complete [code](https://github.com/fuseio/examples/blob/rest-api-guide-trade-v2/trade-api/pages/index.tsx). 💻.
+
```

Comment:
After i copied source code i can se many type errors what should be fixed since we are working on typescript.
![Screenshot 2024-06-02 at 17 11 46](https://github.com/fuseio/fuse-docs/assets/68244817/68cdbc90-a13a-4fad-aab8-f580026977d4)
Also, i tried to use this little app, and it pretty baggy, it requires at least basic error catching and instructions what and where user should put.

https://github.com/fuseio/fuse-docs/assets/68244817/f4ac8185-36d0-41e8-88f0-61c1d41e4511